src/adapter.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Casbin Authors. All Rights Reserved.
```

Comment:
Please do not modify these.

Reply:
Copy from old repo. I will recover these comment.